cpp/include/cudf/detail/stream_compaction.hpp

```diff
@@ -71,6 +71,7 @@ std::unique_ptr<table> drop_duplicates(
   std::vector<size_type> const& keys,
   duplicate_keep_option keep,
   null_equality nulls_equal           = null_equality::EQUAL,
+  null_order null_precedence          = null_order::BEFORE,
```

Comment:
`null_precedence` is not documented in `cudf::detail::drop_duplicates`.
`null_precedence` is not a parameter of `cudf::drop_duplicates`, so it should be documented here.